FirebaseCombineSwift/Sources/Core/CombineSDK.swift

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import FirebaseCore
+import FirebaseCoreInternal
+
+@objc(FIRCombineSDK) private class __CombineSDK: NSObject {}
```

Comment:
This file should not be created. Instead a comparable line to the bottom line should be added for each product's combine implementation.

Reply:
Ok, makes sense.